javascripts/discourse/api-initializers/discourse-topic-list-author.gjs

```diff
@@ -0,0 +1,76 @@
+import { eq } from "truth-helpers";
+import bodyClass from "discourse/helpers/body-class";
+import { withPluginApi } from "discourse/lib/plugin-api";
+import AuthorColumnContent from "../components/author-column-content";
+import AuthorColumnHeader from "../components/author-column-header";
+import CustomCategoriesLatestAvatar from "../components/custom-categories-latest-avatar";
+
+export default {
+  name: "discourse-topic-list-cards",
+  initialize() {
+    withPluginApi("1.39.0", (api) => {
+      api.renderInOutlet(
+        "latest-topic-list-item-topic-poster",
+        CustomCategoriesLatestAvatar
+      );
+
+      const site = api.container.lookup("service:site");
+
+      // TODO: (discourse.hbr-topic-list-overrides) remove the connector rendered below after the legacy topic list is removed from core
+      api.renderInOutlet("above-site-header", <template>
+        {{#if (eq site.useGlimmerTopicList false)}}
+          {{bodyClass "hbr-topic-list__topic-list-author"}}
+        {{/if}}
+      </template>);
+
+      if (site.desktopView) {
```

Comment:
Internally (/t/141133) we're talking about making `site.desktopMode` responsive based on viewport width. If we do that, it'll be possible for it to change at runtime.

Can we easily move this check into the relevant outlet `shouldRender`, or into the transformer callback? That way, this theme component will be future-proofed against that change.

Reply:
Done